Program.cs

```diff
@@ -71,10 +75,9 @@ static async Task MainAsync()
             await roleEvents.ReactOnUserMessage(e, discord);
             await e.Message.RespondAsync("You will receive your Role.");
           }
-          else
+          else if (e.Author.IsBot == false && Blacklist.is_swearword(e.Message.Content.ToLower()))
           {
-            response = messageDistributor.GetMessage(e).ToString();
-            await e.Message.RespondAsync(response);
```

Comment:
Wird wahrscheinlich von @NHGitHub7 wieder eingefuehrt. 🤷‍♂️ Bin auf seine Pull Request gespannt.

Reply:
Müssen mal schauen wie wir das mit dem DcMessageDistributor hinbekommen, den gibt es ja nur weil man die Custom Commands irgendwie ansprechen muss. 
Vielleicht geht das auch über DSharp+, dort muss eben auf eine Column in der Datenbank geprüft und bei einem Treffer reagiert werden.

!CustomCommandName => "CustomCommandName" steht in der Tabelle